test/fnull.js

```diff
@@ -0,0 +1,45 @@
+import { assert } from 'chai';
+
+import * as RA from '../src';
+
+describe('fnull', function() {
+  context('given function and default arguments', function() {
+    specify(
```

Comment:
What would you say if we merged two following tests (except currying test) into one clear block of assertions?
```js
context('given function and default arguments', function() {
   specify('should use argument or appropriate default argument', function() {
      assert.strictEqual(RA.fnull(R.add, [1, 1])(1, 2), 3);
      assert.strictEqual(RA.fnull(R.add, [1, 1])(null, 2), 3);
      assert.strictEqual(RA.fnull(R.add, [1, 1])(null, undefined), 2);
      ....what ever other assertion combinations you can think of using `null` or `undefined` or not nil value...
   });
});
```